### ISSUE TITLE
Double episodes showing as missing in JMM Desktop

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dizzydezz/jmm#433]
```
Double episodes showing as missing in JMM Desktop

http://forum.team-mediaportal.com/threads/my-anime-3.107475/page-49#post-992339

```

Original issue reported on code.google.com by `werndly...@gmail.com` on 10 May 2013 at 12:30

**Comment by GoogleCodeExporter** [comment]
Original comment by `werndly...@gmail.com` on 27 May 2013 at 3:06
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Basically what happened.

The Episode 1 is listed on AniDB as the complete Movie (AID 7089)
This shows up correctly in JMM.

The same file is also listed as being for Episode 1 and 2 of the Area 88 OVA 
(AID 569).

So when the file is imported initially, only the anime for 7089 exists. It does 
not import linked animes (569)
When the 3rd episode is imported, it gains the links from the initial file 
import, but does update the stats on those files to say that files have been 
linked

```

Original comment by `werndly...@gmail.com` on 27 May 2013 at 5:01

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.1.32
```

Original comment by `werndly...@gmail.com` on 27 May 2013 at 7:54
- Changed state: **Fixed**
